README.md

```diff
@@ -213,6 +213,39 @@ spec:
 
 > Note: pay attention to the name of the selector above. You will need to use it to bind the identity to your pod.
 
+If you use the Helm Chart to deploy the custom metrics adapter to your Kubernetes cluster, you can configure Azure AD Pod Identity directly in the config values:
+
+```yaml
+azureAuthentication:
+  # method: {msi,clientSecret,clientCertificate,aadPodIdentity}
+  method: msi
+  # Generate secret file. If false you are responsible for creating secret 
+  # To generate secret file swith to true then fill in values below
+  createSecret: false
+  tenantID: ""
+  clientID: ""
+  clientSecret: ""
+  clientCertificate: ""
+  clientCertificatePath: ""
+  clientCertificatePassword: ""
+  # if you use aadPodIdentity authentication
+  azureIdentityName: "custom-metrics-identity"
+  azureIdentityBindingName: "custom-metrics-identity-binding"
+  # The full Azure resource id of the managed identity (/subscriptions/{SubID}/resourceGroups/{ResourceGroup1}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{IdentityName})
+  azureIdentityResourceId: ""
+  # The Client Id of the managed identity
+  azureIdentityClientId: ""
+```
+
+Switch `method` to `aadPodIdentity` a give the value for the Azure Identity resource id and client id, for example:
```

Comment:
After updating the section above, can change this to something like `An example from the command line...`